bridge_adapters/src/lisp_adapters/numbers.rs

```diff
@@ -44,7 +131,7 @@ impl<'a> SlFromRef<'a, Value> for i32 {
     fn sl_from_ref(value: Value, vm: &'a SloshVm) -> VMResult<i32> {
         match value {
             Value::Int(num) => {
-                let num = from_i56(&num);
+                let num = I56::from_inner(&num);
```

Comment:
I like the I56 thing just because it gives me something I can import that I can then perform I56 type things with. Don't care if we actually use it. More a comment on. There are some i56 types functions. i need them organized in such a way. we could also create a `mod i56 {}`?